### PR TITLE
Use wait_for instead of pause when building DO

### PR DESCRIPTION
**Pull request by DavidWittman**
Rather than twiddling our thumbs for 90 seconds, the `wait_for`
module will poll port 22 until it is ready for connections. This
is faster and more resilient than using the `pause` module here.

Also add *.retry files to the .gitignore

### CODE REVIEW

**Comment by jlund** [comment]
The `wait_for` module is (and was) [already being used](https://github.com/jlund/streisand/blob/master/playbooks/roles/genesis-digitalocean/tasks/main.yml#L35). Your proposed commit adds it for a second time, twenty lines below the original instance :)

Earlier this year, DigitalOcean was failing to accept incoming connections on brand-new droplets in a timely fashion. Despite the OpenSSH daemon reporting that it was ready, connection attempts would just sit there and hang. Notably, this occurred with or without Ansible, and it also happened across multiple regions. The issue (whatever it was) would clear up around a minute later, but the resulting timeouts were completely breaking the playbook.

In other words, the janky 90-second `pause` was added to deal with this DigitalOcean jankiness. Maybe it isn't janky anymore? Let me check.

**Comment by jlund** [comment]
It looks like they fixed it, and I removed the `pause` task. Thanks for bringing it up and reminding me to revisit this step! It's much, much nicer without the artificial delay, and its presence was definitely confusing. I would have made exactly the same assumption that you did.

**Comment by DavidWittman** [comment]
Whoops, apologies for missing that one, Josh. Thanks!
